### PR TITLE
persist query in url

### PR DESCRIPTION
**Pull request by alinauroz**
for #810 

The issue is fixed by keeping the query in url when view is switched on round page

### CODE REVIEW

**Comment by aerugo** [comment]
@alinauroz - we need to resolve some conflicts on this one

**Comment by alinauroz** [comment]
> @alinauroz - we need to resolve some conflicts on this one

resolved

**Comment by aerugo** [comment]
@alinauroz conflict again now after fixing the resolver problems

**Comment by alinauroz** [comment]
I'll resolve

**Comment by alinauroz** [comment]
resolved

File: ui/package.json [hunk 1]
```diff
@@ -117,6 +117,7 @@
     "@babel/core": "7.10.2",
     "@formatjs/cli": "^4.8.3",
     "@neojp/tailwindcss-line-clamp-utilities": "^1.0.1",
+    "@types/cookie-parser": "^1.4.3",
```
Comment (alinauroz) [inline ui/package.json:120]: linter was giving error, types were missing for this module. So i added types

**Pull request merged**